apex-checks/src/main/java/org/fundacionjala/enforce/sonarqube/apex/checks/DmlStatementCheck.java

```diff
@@ -1,66 +0,0 @@
-/*
```

Comment:
Yes, we're not deleting the old checks yet, just separated them into a different package